ozark/pom.xml

```diff
@@ -79,13 +120,37 @@
                     </execution>
                 </executions>
             </plugin>
+            <!--
+             This plugin is reponsible for packaging artifacts
+             as OSGi bundles.  Please refer to
+             http://felix.apache.org/site/apache-felix-maven-bundle-plugin-bnd.html
+             for more information about how to use this plugin.
+           -->
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
```

Comment:
Yes, I mean the latest one at this time. If it doesn't get a constant version anywhere, further changes may break the builds.